src/app/core/theme.component.ts

```diff
@@ -99,6 +114,15 @@ export default class ThemeComponent {
       this.isActive = false;
     },
   };
+
+  // -- MENU --
+  menuService = inject(MenuService);
+  isMenuActive = this.menuService.menu$.pipe(map(menu => menu.isActive));
+  userMockMenu: MenuInputs = {
+    userData: { fullName: 'test user hahahadsdssdsdsdsddh', email: 'test@gmail.com', initials: 'TU' },
+    role: 'admin',
+  };
+
   onToggleChange(isChecked: boolean) {
     console.log(isChecked);
```

Comment:
console.loga nie powinno być

Reply:
to jest `theme.component`, więc console.logi wrzucamy żeby pokazać działanie + ten console.log pochodzi w ogole z innego taska